flow/raster_cache_unittests.cc

```diff
@@ -151,7 +151,11 @@ TEST(RasterCache, DeviceRectRoundOut) {
   canvas.setMatrix(ctm);
   canvas.translate(248, 0);
 
-  cache.Get(*picture, ctm).draw(canvas);
+#ifndef SUPPORT_FRACTIONAL_TRANSLATION
+  canvas.setMatrix(RasterCache::GetIntegralTransCTM(canvas.getTotalMatrix()));
```

Comment:
This line defeats the purpose of this test as stated in the comment above it.

Reply:
If we don't snap, we'd fail the [integral translation assert][1] instead of of bounds assert above. This test tries to reproduce the bounds assert.

[1]: https://github.com/flutter/engine/blob/f9e53c72c656e240a1be969e38e989bb4667878a/flow/raster_cache_key.h#L21

Reply:
Given the current way that we use the raster cache, this whole test doesn't make sense then.

It will be useful when we introduce "cache at 0,0 - render at any fractional translation" later, though. In that case we will see output size differences of up to +1 from the cached image.